examples/manually-instrumented-chatbot/chat-service/Makefile

```diff
@@ -0,0 +1,20 @@
+server:
+	uvicorn chat.app:app --reload
+
+format:
+	ruff check --fix . && ruff format .
+
+typecheck:
+	pyright .
```

Comment:
Kinda curious why you chose pyright here.

Reply:
Arbitrary choice. I can switch to `mypy`.

Reply:
I think `pyright` is faster, though.